src/Utils/__tests__/errors.test.ts

```diff
@@ -1,19 +1,9 @@
 import * as Sentry from "@sentry/browser"
-import { isErrorInfo, sendErrorToService } from "Utils/errors"
+import { sendErrorToService } from "Utils/errors"
 
 jest.mock("@sentry/browser")
 
 describe("errors", () => {
-  describe("#isErrorInfo", () => {
-    it("returns true if an object implements the ErrorInfo interface", () => {
-      const errorInfo = { componentStack: "someString", anotherKey: 0 }
-      expect(isErrorInfo(errorInfo)).toBe(true)
-    })
-    it("returns false if an object does not implement the ErrorInfo interface", () => {
-      const notErrorInfo = { anotherKey: 0 }
-      expect(isErrorInfo(notErrorInfo)).toBe(false)
-    })
-  })
   describe("#sendErrorToService", () => {
```

Comment:
thoughts on adding a separate test case that tests that `setExtra` gets called as many times as there are keys in `errorInfo` object passed in, and restructuring this test case to be focused on expecting `captureException` to be called?

Reply:
Also, I think we should test the `if` on L11 – maybe we can stub `process.env.NODE_ENV` somehow?

Reply:
Yep, that all makes sense to me -- I just deleted these so that the code would compile. I'll add unit tests once there's consensus this is the right direction.